projects/plugins/jetpack/_inc/client/recommendations/prompts/checkbox-answer/index.jsx

```diff
@@ -19,10 +19,16 @@ import { getDataByKey, updateRecommendationsData } from 'state/recommendations';
 import './style.scss';
 
 const CheckboxAnswerComponent = ( { answerKey, checked, info, title, updateCheckboxAnswer } ) => {
-	const toggleCheckbox = useCallback( () => {
-		const newCheckedValue = ! checked;
-		updateCheckboxAnswer( { [ answerKey ]: newCheckedValue } );
-	}, [ answerKey, checked, updateCheckboxAnswer ] );
+	const toggleCheckbox = useCallback(
+		e => {
+			if ( e.target.type !== 'checkbox' ) {
+				return;
+			}
```

Comment:
Clicking on a `<label>` also triggers this event handler and we don't want it to run twice. We only want to run it when the target is of type `checkbox`.